substrate/rpc/src/subscriptions.rs

```diff
@@ -34,12 +34,12 @@ type Id = u64;
 pub struct Subscriptions {
 	next_id: AtomicUsize,
 	active_subscriptions: Mutex<HashMap<Id, oneshot::Sender<()>>>,
-	event_loop: Remote,
+	event_loop: TaskExecutor,
```

Comment:
why?

Reply:
I just thought that `event_loop` term is more related to what happens to be [reactor](https://docs.rs/tokio/0.1.7/tokio/reactor/index.html) now. And now it seems we submit futures/tasks to the executor rather than to an event loop.